addons/mail/models/discuss/discuss_channel_member.py

```diff
@@ -37,7 +37,8 @@ class ChannelMember(models.Model):
         help="All Messages if not specified",
     )
     mute_until_dt = fields.Datetime("Mute notifications until", help="If set, the member will not receive notifications from the channel until this date.")
-    is_pinned = fields.Boolean("Is pinned on the interface", default=True)
+    is_pinned = fields.Boolean("Is pinned on the interface", compute="_compute_is_pinned", search="_search_is_pinned")
+    unpin_dt = fields.Datetime("Unpin date", help="Contains the date and time when the channel was unpinned by the partner.")
```

Comment:
probably need to index those fields (probably combined index with other fields as well)?